modules/flowable-cmmn-rest/src/main/java/org/flowable/cmmn/rest/service/api/history/task/HistoricTaskInstanceBaseResource.java

```diff
@@ -218,6 +218,12 @@
             }
         }
 
+        if (request.getIncludeProcessVariables() != null) {
```

Comment:
I think `request` should be `queryRequest` instead.

Reply:
Yes it should 2230030